### PR TITLE
BUG: fix DateTime type issue

### PR DESCRIPTION
**Pull request by anish-mudaraddi**
Really simple fix for a type issue with DateTime Actions - previously using float for `seconds` - will default to `int` I don't think anyone will want to write a query using fractions of a second :)

This fix is required #125 to be merged first

### Special Notes:

---

### Submitter:

Have you (where applicable):

* [ ] Added unit tests?
* [ ] Checked the latest commit runs on Dev?
* [ ] Updated the example config file(s) and README?

---

### Reviewer

Does this PR:

* [ ] Place non-StackStorm code into the `lib` directory?
* [ ] Have unit tests for the action/sensor and `lib` layers?
* [ ] Have clear and obvious action parameter names and descriptions?

### CODE REVIEW

**Review by meoflynn — APPROVED** [review]
LGTM - ready to merge once #125 has been merged

**Review by jacob-ward — APPROVED** [review]

**Pull request merged**
